### ISSUE TITLE
Error when retrieving postgres query results containing a numeric column

### ISSUE THREAD

**Issue by raskchanky** [paupino/rust-decimal#194]
Hi,

I'm using the `postgres` feature of your crate, but for reasons I don't understand, it fails to compile, saying that `FromSql` isn't implemented.  Here is an incomplete code sample that illustrates my use:

```rust
use rust_decimal::Decimal;

struct Widget {
    id:          i64,
    amount:      Decimal,
}

for row in &client.query("SELECT id, amount FROM widgets", &[]).unwrap() {
  let w = Widget {
                        id: row.get(0),
                        amount: row.get(1),
                    };
}
```

If I try to compile this, I get the following error:

```
error[E0277]: the trait bound `rust_decimal::decimal::Decimal: tokio_postgres::types::FromSql<'_>` is not satisfied
  --> src/main.rs:46:37
   |
46 |                         amount: row.get(1),
   |                                     ^^^ the trait `tokio_postgres::types::FromSql<'_>` is not implemented for `rust_decimal::decimal::Decimal`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
```

My `Cargo.toml` has this in it:

```toml
[dependencies]
postgres = { version = "0.16.0-rc.1", features = ["with-uuid-0_7", "with-chrono-0_4"] }
rust_decimal = { version = "1.0.1", features = ["postgres"] }
```

I'd be really grateful for any ideas you might have on what I'm doing wrong here.  Thanks!

**Comment by paupino** [comment]
My initial hunch is it's the new Tokio rewrite that is happening as part of `postgres` `0.16`. I'll need to update support within Rust Decimal to support the new release candidate - all in all, shouldn't be too hard to implement!
I'm hoping to block some real time to focus on the project over the weekend so hopefully I can knock out a number of these issues.

**Comment by paupino** [comment]
This should be fixed with #206 - I'll see if I can get a release out for this soon.

**Comment by paupino** [comment]
Released with `1.1.0`
